src/DataDrive.ts

```diff
@@ -90,8 +90,8 @@ export class DataDrive {
   public async put(
     filename: string,
     content: Buffer | NodeJS.ReadableStream | string,
+    id: string = randomUUID(),
```

Comment:
can you make it an options object? I don't want to do a breaking change if we add other options later

Reply:
good idea, thanks!